modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/resources.jsp

```diff
@@ -35,6 +35,34 @@ String inlineEditSaveURL = GetterUtil.getString((String)request.getAttribute(CKE
 	long javaScriptLastModified = PortalWebResourcesUtil.getLastModified(PortalWebResourceConstants.RESOURCE_TYPE_EDITOR_CKEDITOR);
 	%>
 
+	<script data-senna-track="temporary" type="text/javascript">
+		window.CKEDITOR_GETURL = function (resource) {
+			var languageParameter = '&languageId=' + themeDisplay.getLanguageId();
+
+			// If this is not a full or absolute path.
+
+			if (resource.indexOf(':/') == -1 && resource.indexOf('/') !== 0) {
+				resource = this.basePath + resource;
+			}
+
+			// Add the timestamp, except for directories.
+
+			if (
+				this.timestamp &&
```

Comment:
One other question I forgot to mention. What is `this` when the function is called? Seems more robust and explicit to call out that it is `window.timestamp` (if `window` is really what `this` is).

Love dem globals!